packages/overlay/src/VirtualTrigger.ts

```diff
@@ -41,5 +40,6 @@ export class VirtualTrigger implements VirtualElement {
                 return;
             },
         };
+        return boundingRect;
```

Comment:
I've seen a few instances of this pattern now:

```ts
const name = value;
return name;
```

which makes me think it's an intentional thing and there's something I'm missing.

Reply:
I use this pattern because it's debuggable without code flow edits...can take it or leave it. Once things are working, that is 😉